docs/src/main/sphinx/release/release-444.md

```diff
@@ -0,0 +1,97 @@
+# Release 444 (2 Apr 2024)
+
+## General
+
+* Fix failure for queries containing large numbers of `LIKE` terms in boolean
+  expressions. ({issue}`21235`)
+
+## Docker image
+
+* Update Java runtime to Java 22. ({issue}`21161`)
+
+## BigQuery connector
+
+* Fix failure when reading BigQuery views with [Apache
+  Arrow](https://arrow.apache.org/docs/). ({issue}`21337`)
+
+## ClickHouse connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Delta Lake connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned ACLs with the native S3 file system. ({issue}`21176`)
+* Add concurrent write reconciliation for non-blind `INSERT`. ({issue}`20983`)
+* Add support for reading tables with [v2
+  checkpoints](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#v2-spec).
+  ({issue}`19345`)
+* {{breaking}} Remove support for split size configurataion the catalog
+  properties `delta.max-initial-splits` and `delta.max-initial-split-size` and
+  the catalog session property `max_initial_split_size`. ({issue}`21320`)
+
+## Druid connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Hive connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned ACLs with the native S3 file system. ({issue}`21176`)
+
+## Hudi connector
+
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned ACLs with the native S3 file system. ({issue}`21176`)
+
+## Iceberg connector
+
+* Add support for the `metadata_log_entries` system table. ({issue}`20410`)
+* Add support for reading `BYTE_STREAM_SPLIT` encoding in Parquet files. ({issue}`8357`)
+* Add support for canned ACLs with the native S3 file system. ({issue}`21176`)
+
+## Ignite connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## MariaDB connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## MySQL connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Oracle connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## PostgreSQL connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Redshift connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## SingleStore connector
+
+* Improve performance of reading table comments.  ({issue}`21161`)
+
+## Snowflake connector
+
+* Add support for creating tables with table comments. ({issue}`21305`)
+* Add support for commenting on the existing tables. ({issue}`21305`)
+* Add support for top-N pushdown. ({issue}`21219`)
```

Comment:
Should this and other pushdown changes be framed as performance improvements?

Reply:
It could

Reply:
There's no such a thing as "top N" from a language perspective. This should be phrased as relating to queries that involve "ORDER BY" and "LIMIT". We had similar items in older releases, so please look them up.

Reply:
Updated .. but now its definitely inconsistent with the others.

Reply:
FYI, we have `Top-N pushdown` section in https://trino.io/docs/current/optimizer/pushdown.html#topn-pushdown

Reply:
Yeah .. that is the name we chose for it... we could certainly link to it from release notes to make it simpler